tests/plugins/org.polarsys.capella.test.diagram.tools.ju/src/org/polarsys/capella/test/diagram/tools/ju/xab/DragAndDropPhysicalComponent.java

```diff
@@ -25,18 +42,18 @@ public void test() throws Exception {
     Session session = getSession(getRequiredTestModel());
     SessionContext context = new SessionContext(session);
 
-    testOnXAB(context, PA__PAB_DIAGRAM, PA__PAB_DIAGRAM,
-        PA__PC_PART_PC14);
-    // drag and drop sub physical component
-    testOnXAB(context, PA__PAB_DIAGRAM, PA__PAB_COMPONENT_PC2,
-        PA__PC_PART_PC2_1);
+    oldTest(context);
+    newTest(session, context);
+  }
 
+  public void oldTest(SessionContext context) {
```

Comment:
does old tests are deprecated? shall be removed?
otherwise, can you rename these methods oldTests/newTests to something like "non deploy / deploy"